build/build.xml

```diff
@@ -241,6 +241,79 @@
 		</sequential>
 	</macrodef>
 
+	<macrodef name="build-addon-without-help-indexes" description="build the specified addon">
```

Comment:
Don't! ;)
Ant build will not be used much more time (hopefully), so not worth spending your time with this refactoring.